src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/StaffRefDataController.java

```diff
@@ -323,4 +324,29 @@ public ResponseEntity<StaffProfileCreationResponse> createStaffUserProfile(@Requ
         return ResponseEntity.status(HttpStatus.CREATED).body(staffProfileCreationResponse);
     }
 
+
+
+    @PutMapping(
+            consumes = APPLICATION_JSON_VALUE,
+            produces = APPLICATION_JSON_VALUE,
+            path = {"/profile"}
+    )
+    @Secured("staff-admin")
+    @ResponseBody
+    @ResponseStatus(value = HttpStatus.CREATED)
+    @Transactional
```

Comment:
Can you please add swagger annotations for Authorisation and response?

Reply:
done